src/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -188,4 +188,16 @@
   <data name="Arg_PathEmpty" xml:space="preserve">
     <value>The path is empty.</value>
   </data>
+  <data name="ArgumentOutOfRange_NeedPosNum" xml:space="preserve">
+    <value>Positive number required.</value>
+  </data>
+  <data name="Argument_InvalidAppendMode" xml:space="preserve">
+    <value>Append access can be requested only in write-only mode.</value>
```

Comment:
Where is this resource used?

Reply:
Apparently it's not used. Thanks for catching it. I removed it.